app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -52,16 +51,47 @@ def perform(form526_submission_id, full_name)
       end
     end
 
+    # class AccountNotFoundError < RuntimeError; end
+
     private
 
-    def bp_readings(client)
-      @bp_readings ||= client.list_resource('observations')
-      @bp_readings.present? ? FastTrack::HypertensionObservationData.new(@bp_readings).transform : []
-    end
+    class LighthouseVetsHealth
```

Comment:
I do like the idea of this work staying in the job file but I don't think this class inside a class is going to fly with pr review. If you want a nice reusable object I could see this being a struct that gets passed the submission and the account.

That way it's just a small object that returns formatted data. 🤷  maybe? Thoughts?

Reply:
I didn't pull the class out into a separate file because no other code uses it. The class exists to encapsulate Lighthouse-related methods and clearly distinguish the boundary between it and other methods that send emails, upload pdf, and set special issues. I don't see it as a struct, as in it's not simply a data object -- it's a Lighthouse client that uses an `Account` DB record and parses Lighthouse results.

Reply:
I totes hear you and I don't disagree BUT - I know stylistically the reviewers that need to approve this (not our team) wont let it get merged like that. I think it's a very rails-style thing that they like to see all classes in their own files with their own unit specs around them.